apps/src/templates/instructions/InstructionsCSF.jsx

```diff
@@ -397,79 +336,43 @@ class TopInstructions extends React.Component {
     const leftColHeight = minIconHeight;
     const middleColHeight = minInstructionsHeight;
     const rightColHeight = collapseButtonHeight + scrollButtonsHeight;
-
-    // Only include resizer height if resizer is available
-    const resizerHeight = collapsed ? 0 : RESIZER_HEIGHT;
```

Comment:
why are we removing `resizerHeight` here? i would think we would want to keep this, and continue adding `resizerHeight` below

Reply:
we are still adding the resizer height - now we are just adding it all the time with the constant. Before the CSF instructions used to not show the resizer bar in certain cases but to be more consistent with CSD/CSP I did away with that.

Reply:
ohh, okay. yay for standardizing!